frontends/web/src/routes/new-settings/components/advanced-settings/enable-tor-proxy-setting.module.css

```diff
@@ -0,0 +1,13 @@
+.status {
+    color: var(--color-secondary);
+    font-size: 14px;
+    margin: 0;
+    margin-right: var(--space-quarter);
+    overflow: hidden;
+    text-overflow: ellipsis;
+}
+
+.container {
+    display: flex;
+    align-items: center;
+}
```

Comment:
is that the only component that uses text+ChevronRightDark, else maybe it's worth having an other prop for the text in SettingsItem and moving this CSS to SettingsItem.

Reply:
I was thinking about this. I think it makes sense to create a new prop to achieve a result similar to that.  E.g lets call it `displayedValue`. The reason is bc I wanna keep the `ExtraComponent` prop, since in some setting items there'll be text+**icon** (not just text+Chevron right).

I'll make a change for now n  feel free to lmk what you think:)